Mathlib/Order/SetNotation.lean

```diff
@@ -299,4 +299,3 @@ theorem iSup_eq_iUnion (s : ι → Set α) : iSup s = iUnion s :=
 theorem iInf_eq_iInter (s : ι → Set α) : iInf s = iInter s :=
   rfl
 #align set.infi_eq_Inter Set.iInf_eq_iInter
-
```

Comment:
is that an accident?

Reply:
This is the vscode auto-save behavior; many files have stray trailing newlines if they were ported by people not using vscode.